lesson_5/index5.html

```diff
@@ -69,5 +74,47 @@ <h1>Lesson_4</h1>
                 <td>0</td>
             </tr>
         </table>
+        <br>
+        <br>
+        <br>
+        <caption>Table 2</caption>
+    <table border="=1">
+        <tr>
+            <th rowspan="2">Heading</th>
+            <th>Student</th>
+            <th colspan="3">Details</th>
+        </tr>
+        <tr>
+            <th>Id</th>
+            <th>Name</th>
+            <th>Department</th>
+            <th>Roll Namber</th>
+        </tr>
+        <tr>
+            <td rowspan="4">Student List</td>
+            <td>1</td>
+            <td>Victor</td>
+            <td>Computer Science</td>
+            <td>12345</td>
+        </tr>
+        <tr>
+            <td>2</td>
+            <td>Williams</td>
+            <td>Electronics</td>
+            <td>23456</td>
+        </tr>
+        <tr>
+            <td>3</td>
+            <td>Harry</td>
+            <td>Electrical</td>
+            <td>34567</td>
+        </tr>
+        <tr>
+            <td>4</td>
+            <td>Rick</td>
+            <td>Civil</td>
+            <td>45678</td>
+        </tr>
+    </table>
```

Comment:
table tag is inside of body, so should be nested by spaces as all tags

Reply:
ok